common/persistence/visibility/store/sql/query_converter.go

```diff
@@ -604,6 +605,27 @@ func (c *QueryConverter) parseSQLVal(
 		return status, nil
 	}
 
+	if saName == searchattribute.ExecutionDuration {
+		if durationStr, isString := value.(string); isString {
+			// To support durations passed as golang durations such as "300ms", "-1.5h" or "2h45m".
+			// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+			// Custom timestamp.ParseDuration also supports "d" as additional unit for days.
```

Comment:
Do you know what "d" is not supported by standard library? Because its length may vary :open_mouth:.

Reply:
> Because its length may vary 😮.

What do you mean?